src/context/directory/handlers/roles.js

```diff
@@ -34,12 +34,6 @@ async function dump(context) {
   roles.forEach((role) => {
     const roleFile = path.join(rolesFolder, sanitize(`${role.name}.json`));
     log.info(`Writing ${roleFile}`);
-
-    // remove empty description
-    if (role.description === null) {
-      delete role.description;
-    }
-
```

Comment:
May need to be restored.

Reply:
correct, master has https://github.com/auth0/auth0-deploy-cli/blob/master/src/context/directory/handlers/roles.js